corehq/apps/repeaters/migrations/f_0001_adjust_auth_field_format.py

```diff
@@ -0,0 +1,37 @@
+from django.db import migrations
+
+from corehq.apps.repeaters.models import Repeater
+from corehq.sql_db.operations import HqRunPython
+from corehq.util.couch import iter_update, DocUpdate
+
+
+def migrate_auth_field(apps, schema_editor):
+    repeater_ids = [row['id'] for row in Repeater.view(
+        'receiverwrapper/repeaters',
+        include_docs=False,
+        reduce=False,
+        wrap_doc=False
+    )]
+    iter_update(
+        db=Repeater.get_db(),
+        fn=migrate_repeater,
+        ids=repeater_ids,
+    )
+
+
+def migrate_repeater(repeater_doc):
+    if "use_basic_auth" in repeater_doc:
+        use_basic_auth = repeater_doc['use_basic_auth'] == True
+        del repeater_doc['use_basic_auth']
+        if use_basic_auth:
+            repeater_doc["auth_type"] = "basic"
+        return DocUpdate(repeater_doc)
```

Comment:
i'm not too familiar with how this process runs, but is it ok for this function to return None when the repeater_doc doesn't have a key for use_basic_auth?